ci/jenkins/jobs/projects.yaml

```diff
@@ -237,6 +255,88 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: whole-conformance-pending-label
+          node: null
+          description: 'This is for marking PR as pending for whole conformance test.'
+          builders:
+            - builder-pending-label
+          trigger_phrase: null
+          allow_whitelist_orgs_as_admins: false
+          admin_list: []
+          org_list: []
+          white_list: []
+          only_trigger_phrase: false
+          trigger_permit_all: true
+          status_context: jenkins-whole-conformance
+          status_url: --none--
+          success_status: Pending test. Mark as failure. Check not required. Add comment /test-whole-conformance to trigger.
+          failure_status: Pending test. Mark as failure. Check not required. Add comment /test-whole-conformance to trigger.
+          error_status: Pending test. Mark as failure. Check not required. Add comment /test-whole-conformance to trigger.
+          triggered_status: null
+          started_status: null
+          wrappers: []
+          publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: whole-conformance
+          node: 'antrea-test-node'
+          description: 'This is the {test_name} test for {name}.'
+          builders:
+            - builder-workload-cluster-setup
+            - builder-prepare-antrea
+            - builder-conformance:
+                mode_regex: 'certified-conformance'
+                focus_regex: ''
+                skip_regex: ''
+            - builder-workload-cluster-cleanup
+          trigger_phrase: .*/test-(whole-conformance|all).*
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: false
+          status_context: jenkins-whole-conformance
+          status_url: null
+          success_status: Build finished.
+          failure_status: Failed. Check not required. Add comment /test-whole-conformance to re-trigger.
+          error_status: Failed. Check not required. Add comment /test-whole-conformance to re-trigger.
```

Comment:
I think we can remove the explicit "Check not required" here. If someone decided to run the test and it failed, then it probably needs to be addressed, even if we do not actually block the PR.

Reply:
Yes, it makes sense. Updated.